agreements/agreements-service.yaml

```diff
@@ -557,9 +574,19 @@ components:
                 example: "product.priceRises"
         evaluationType:
           type: string
-          enum: [equal , greater, less, complex]
+          description: How the rule should be evaluated -<br>
```

Comment:
Could replace the html tags here with markdown, e.g.

```
          description: | 
            How the rule should be evaluated
            * equal = rule is true if the lotAttribute is equal to the transactionData
            * greater = rule is true if the lotAttribute is greater than the transactionData
```
According to the OpenAPI [spec](https://swagger.io/specification/), tooling must support markdown as a minimum (Swagger Editor now does)